wagtail/images/tests/tests.py

```diff
@@ -359,6 +359,9 @@ def test_get(self):
         self.assertEqual(response.status_code, 200)
         self.assertTrue(response.streaming)
         self.assertEqual(response["Content-Type"], "image/png")
+        for _byte in response:
+            # Ensure the file can actually be read
+            self.assertTrue(_byte)
```

Comment:
Nit: Could we do a better assertion than just the truthyness of the bytes? Perhaps passing it into `willow` and checking it looks like a sensible image?

I realise that's slightly out of scope of this PR, but it'd be good to confirm what we get back is actually an image and not garbage data.

Similarly, it'd then be great to add similar assertions to the related test cases.